Makefile.dep

```diff
@@ -191,6 +186,11 @@ ifneq (,$(filter gnrc_netif,$(USEMODULE)))
   USEMODULE += netif
   USEMODULE += l2util
   USEMODULE += fmt
+
+  # Don't register netdevs if there is only a single one of them
+  ifeq (,$(filter gnrc_netif_single,$(USEMODULE)))
+    USEMODULE += netdev_register
+  endif
```

Comment:
netdev registration is independent of `gnrc` though.
We should still get the same MAC addresses with LWIP or OpenThread.

Reply:
Ok, I understand now the rationale behind it. But I think this module should be pulled when there's a user. It should at least check if there's someone trying to pull `netdev`. Now this should be possible with this PR

Reply:
Done! (see https://github.com/RIOT-OS/RIOT/pull/13565/commits/ad61f2852da2eb15bc0fcddab1ffe7203e3f6c92)

Reply:
Looks good to me!